src/controllers/words.ts

```diff
@@ -162,6 +163,6 @@ export const createWord = async (data, connection) => {
   /* Wait for all the Examples to be created and then add them to the Word document */
   const resolvedExamples = await Promise.all(savedExamples);
   const exampleIds = getDocumentsIds(resolvedExamples);
-  newWord.examples = exampleIds;
+  (newWord as any).examples = exampleIds;
```

Comment:
we don't want to use `any` please use `unknown` if you're unable to use a more specific type